Mathlib/RingTheory/PowerSeries/PiTopology.lean

```diff
@@ -156,6 +156,28 @@ theorem summable_of_tendsto_order_atTop_nhds_top [LinearOrder ι] [LocallyFinite
   contrapose! hk
   exact coeff_of_lt_order _ <| by simpa using (hi k hk.le)
 
+variable {R} in
+/-- The geometric series converges if the constant term is zero. -/
+theorem summable_pow_of_constantCoeff_eq_zero {f : PowerSeries R} (h : f.constantCoeff = 0) :
+    Summable (f ^ ·) :=
+  MvPowerSeries.WithPiTopology.summable_pow_of_constantCoeff_eq_zero h
+
+section GeomSeries
+variable {R : Type*} [TopologicalSpace R] [Ring R] [IsTopologicalRing R] [T2Space R]
+variable {f : PowerSeries R}
+
+/-- Formula for geometric series. -/
```

Comment:
These theorems should not need T2Space.

Reply:
Resolving for the same reason as the previous one